install/_min-requirements.sh

```diff
@@ -1,6 +1,6 @@
 # Don't forget to update the README and othes docs when you change these!
 MIN_DOCKER_VERSION='19.03.6'
-MIN_COMPOSE_VERSION='1.28.0'
+MIN_COMPOSE_VERSION='2.0.0'
```

Comment:
Let's have this match exactly (including patch version) the min version in our test matrix. Does 2.0.0 really work? 🤔 If not, update the test matrix (good way to find out btw ;), else update this to 2.0.1.

Reply:
Yep, it exists https://github.com/docker/compose/releases/tag/v2.0.0

Reply:
ah looks like there's a reason why we're using 2.0.1 here
https://stackoverflow.com/questions/69524003/cannot-install-docker-compose-v2-invalid-plugin-exec-format-error